client/scss/components/_footer.scss

```diff
@@ -112,6 +112,11 @@
   display: block;
   text-decoration: none;
   border-left: 1px solid color('charcoal');
+  transition: color 200ms ease;
```

Comment:
I was wondering about this, too – quite possibly, yeah. Perhaps a PR to rationalise them throughout.